exercises/java/encryption-context-complete/src/test/java/sfw/example/esdkworkshop/AppTest.java

```diff
@@ -0,0 +1,14 @@
+package sfw.example.esdkworkshop;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+/** Unit test for simple App. */
+public class AppTest {
+  /** Rigourous Test :-) */
```

Comment:
This comment might not be understood well by all people doing this workshop.

Reply:
This was autogenerated by Maven's "generate me a Maven project" and will be going away. :)